ios/Classes/BetterPlayer.h

```diff
@@ -38,8 +38,6 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic) float playerRate;
 @property(nonatomic) int overriddenDuration;
 @property(nonatomic) AVPlayerTimeControlStatus lastAvPlayerTimeControlStatus;
-@property(nonatomic) bool isBackToAppButtonPressed;
```

Comment:
memo
This variable is added on https://github.com/dwango-nfc/betterplayer/pull/42/files#diff-a2f4ffe7e3d54263623c9291e48051d1a11bd712ca7ca05d00cee0338e619598R41
It is related to DAF-4278.